rnote-ui/src/appwindow/mod.rs

```diff
@@ -610,6 +610,11 @@ impl RnAppWindow {
             "format-borders",
             &format.show_borders.to_variant(),
         );
+        adw::prelude::ActionGroupExt::change_action_state(
+            self,
+            "block-pinch-zoom",
+            &active_tab.block_pinch_zoom().to_variant(),
+        );
```

Comment:
we could do it like this and it works, because the property `block-pinch-zoom` is updated here when the active tab changes. But I would actually prefer it if this would be a property binding, similar to the `touch-drawing` property.

Meaning:  
- also add a "global" `block-pinch-zoom` property on the appwindow
- create a binding to the canvaswrapper property in `canvaswrapper.init_reconnect()`
- refactor the action so that is a property-action of the appwindow `block-pinch-zoom` property.